dali/operators/image/color/color_space_conversion_test.cc

```diff
@@ -56,4 +59,11 @@ TYPED_TEST(ColorSpaceConversionToYCbCrTest, test) {
   this->RunTest("ColorSpaceConversion", nullptr, 0, false, 0.002);
 }
 
+typedef ::testing::Types<RGB, BGR, Gray, YCbCr> ConvertibleToLab;
```

Comment:
I think you should extend ColorSpaceConversionToRGBTest and ColorSpaceConversionToBGRTest as well by adding `Lab` to tested input types.

Reply:
had add conversion test between Lab and others